### PR TITLE
Update migrations

### LINKED ISSUE #560: Migrations article outdated
https://documentation.platform-os.com/tutorials/migrations/migrating-data

Looks like a lot has changed. IE. migrations are run on deploy, so run command is not needed.

### PR DESCRIPTION
**Pull request by ignacy**
Fixes https://github.com/mdyd-dev/nearme-documentation/issues/560

Add information about `marketplace-kit migrations list` and some more explenation.

### CODE REVIEW

File: marketplace_builder/views/pages/tutorials/migrations/migrating-data.liquid [hunk 1]
```diff
@@ -72,6 +78,10 @@ marketplace-kit migrations run local 20180909123423
 
 A newly created migration is in pending status for as long as it has not been run. Once it's executed (either through a deploy or using the `migrations run` command) it progresses to either done (on success) or error status.
 
+You can check status of all migrations using `marketplace-kit migrations list` command.
+
 Once migration is in done status it can't be run any more - this makes it safer to run the same queue of migrations on all environments.
 
 When an error occurs, the migration can be edited and rerun until it succeeds.
+
+Although migrations are run in backnground worker and will not affect performence of your site we encourage using <a href="/tutorials/data-import-export/import">Data Import/Export</a> for any larger data processing jobs. Migrations are best for seeding data and doing small changes to model schema.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/migrations/migrating-data.liquid]:
```suggestion
Although migrations are run in background worker and will not affect performence of your site we encourage using <a href="/tutorials/data-import-export/import">Data Import/Export</a> for any larger data processing jobs. Migrations are best for seeding data and doing small changes to model schema.
```

File: marketplace_builder/views/pages/tutorials/migrations/migrating-data.liquid [hunk 2]
```diff
@@ -72,6 +78,10 @@ marketplace-kit migrations run local 20180909123423
 
 A newly created migration is in pending status for as long as it has not been run. Once it's executed (either through a deploy or using the `migrations run` command) it progresses to either done (on success) or error status.
 
+You can check status of all migrations using `marketplace-kit migrations list` command.
+
 Once migration is in done status it can't be run any more - this makes it safer to run the same queue of migrations on all environments.
 
 When an error occurs, the migration can be edited and rerun until it succeeds.
+
+Although migrations are run in background worker and will not affect performence of your site we encourage using <a href="/tutorials/data-import-export/import">Data Import/Export</a> for any larger data processing jobs. Migrations are best for seeding data and doing small changes to model schema.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/migrations/migrating-data.liquid]:
```suggestion
Although migrations are run in background worker and will not affect performance of your site we encourage using <a href="/tutorials/data-import-export/import">Data Import/Export</a> for any larger data processing jobs. Migrations are best for seeding data and doing small changes to model schema.
```

File: marketplace_builder/views/pages/tutorials/migrations/migrating-data.liquid [hunk 3]
```diff
@@ -55,9 +61,9 @@ For example, the following mutation can be put into the file created by the comm
 
 {% include 'alert/note', content: 'You can use deploy or sync, but the migration you want to execute has to be delivered to the target environment first.' %}
 
-Deployment will upload and try to run migrations one by one (ordered by timestamp) and it will fail if one of the migrations fails.
+Deployment will upload and <b>try to run migrations one by one (ordered by timestamp)</b> and it will fail if one of the migrations fails. In modules context, migrations are run when the deployment is done through Partner Portal or using the `marketplace-kit deploy -p` option.
 
-The `marketplace-kit migrations run` command runs a single migration:
+The `marketplace-kit migrations run` command starts a backnground process that runs a single migration:
```
Comment (pavelloz) [inline marketplace_builder/views/pages/tutorials/migrations/migrating-data.liquid]:
```suggestion
The `marketplace-kit migrations run` command starts a background process that runs a single migration:
```

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited! Thank you!

**Pull request merged**